gourmet/backends/db.py

```diff
@@ -1867,8 +1867,9 @@ def parse_ingredient (self, s, conv=None, get_key=True):
         debug('ingredient_parser handed: %s'%s,0)
         # Strip whitespace and bullets...
         d={}
-        s = s.decode('utf8').strip(
-            '\u2022\u2023\u2043\u204C\u204D\u2219\u25C9\u25D8\u25E6\u2619\u2765\u2767\u29BE\u29BF\n\t #*+-')
+        if isinstance(s, bytes):
+            s = s.decode('utf8').strip(
+                '\u2022\u2023\u2043\u204C\u204D\u2219\u25C9\u25D8\u25E6\u2619\u2765\u2767\u29BE\u29BF\n\t #*+-')
```

Comment:
don't we need to strip this stuff as well, if it already is a `str` instance?

Reply:
thanks